cbmc.inc

```diff
@@ -8,7 +8,9 @@ TOOL_NAME=CBMC
 run()
 {
   if [ "$PROP" = "termination" ] ; then
-    PROPERTY="$PROPERTY --no-assertions"
+    EC=42
+    echo "EC=$EC" >> $LOG.ok
+    return
```

Comment:
Question: do we do Termination in CBMC or not? I suppose it might be too risky at this point, so I'm leaning towards supporting this change.